### PR TITLE
Adding vault 0.8.0-rc1

### PR DESCRIPTION
**Pull request by chrishoffman**

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
warning: insecure protocol git:// detected: git://github.com/hashicorp/docker-vault
diff --git a/_bashbrew-arches b/_bashbrew-arches
index f47c3d0..defdb60 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -3,4 +3,5 @@ vault:0.6.5 @ amd64
 vault:0.7.0 @ amd64
 vault:0.7.2 @ amd64
 vault:0.8.0-beta1 @ amd64
+vault:0.8.0-rc1 @ amd64
 vault:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 3af5f31..bade469 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -4,4 +4,5 @@ vault:0.7.0
 vault:0.7.2
 vault:0.7.3
 vault:0.8.0-beta1
+vault:0.8.0-rc1
 vault:latest
diff --git a/vault_0.7.0/Dockerfile b/vault_0.8.0-rc1/Dockerfile
similarity index 99%
copy from vault_0.7.0/Dockerfile
copy to vault_0.8.0-rc1/Dockerfile
index 7c6c502..e9db228 100644
--- a/vault_0.7.0/Dockerfile
+++ b/vault_0.8.0-rc1/Dockerfile
@@ -2,7 +2,7 @@ FROM alpine:3.6
 MAINTAINER Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
 
 # This is the release of Vault to pull in.
-ENV VAULT_VERSION=0.7.0
+ENV VAULT_VERSION=0.8.0-rc1
 
 # This is the release of https://github.com/hashicorp/docker-base to pull in order
 # to provide HashiCorp-built versions of basic utilities like dumb-init and gosu.
diff --git a/vault_0.7.0/docker-entrypoint.sh b/vault_0.8.0-rc1/docker-entrypoint.sh
similarity index 100%
copy from vault_0.7.0/docker-entrypoint.sh
copy to vault_0.8.0-rc1/docker-entrypoint.sh
```

</details>

File: library/vault [hunk 1]
```diff
@@ -6,4 +6,5 @@
 0.7.2: git://github.com/hashicorp/docker-vault@02af9417ae42bf6290b4b36c2188b1836f1017a8 0.7.2
 0.7.3: git://github.com/hashicorp/docker-vault@02af9417ae42bf6290b4b36c2188b1836f1017a8 0.7.3
 0.8.0-beta1: git://github.com/hashicorp/docker-vault@20c626ab1c89a17c24f370ae18cfe6cbc0eb8d33 0.8.0-beta1
```
Comment (yosifkit) [inline library/vault]: Should the beta version be dropped?

Comment (chrishoffman) [inline library/vault]: I can drop the beta.  I thought it didn't get removed if you removed it from this file, so it didn't really matter.

Comment (yosifkit) [inline library/vault]: It'll still show up in the "Supported tags" list on the Docker Hub and get rebuilt if the base image gets updated.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index f47c3d0..0741706 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -2,5 +2,5 @@ vault:0.6.4 @ amd64
 vault:0.6.5 @ amd64
 vault:0.7.0 @ amd64
 vault:0.7.2 @ amd64
-vault:0.8.0-beta1 @ amd64
+vault:0.8.0-rc1 @ amd64
 vault:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 3af5f31..9507bf2 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -3,5 +3,5 @@ vault:0.6.5
 vault:0.7.0
 vault:0.7.2
 vault:0.7.3
-vault:0.8.0-beta1
+vault:0.8.0-rc1
 vault:latest
diff --git a/vault_0.8.0-beta1/Dockerfile b/vault_0.8.0-rc1/Dockerfile
similarity index 99%
rename from vault_0.8.0-beta1/Dockerfile
rename to vault_0.8.0-rc1/Dockerfile
index ae5c15e..e9db228 100644
--- a/vault_0.8.0-beta1/Dockerfile
+++ b/vault_0.8.0-rc1/Dockerfile
@@ -2,7 +2,7 @@ FROM alpine:3.6
 MAINTAINER Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
 
 # This is the release of Vault to pull in.
-ENV VAULT_VERSION=0.8.0-beta1
+ENV VAULT_VERSION=0.8.0-rc1
 
 # This is the release of https://github.com/hashicorp/docker-base to pull in order
 # to provide HashiCorp-built versions of basic utilities like dumb-init and gosu.
diff --git a/vault_0.8.0-beta1/docker-entrypoint.sh b/vault_0.8.0-rc1/docker-entrypoint.sh
similarity index 100%
rename from vault_0.8.0-beta1/docker-entrypoint.sh
rename to vault_0.8.0-rc1/docker-entrypoint.sh
```

</details>

**Comment by yosifkit** [comment]
Build test of #3266; d01d7c03 (`vault`):

```console

$ bashbrew build vault:0.6.4
Using bashbrew/cache:6f73c64702e812f31c1508217e001ae8274b77e86c5a36e87b911836c99ccfa5 (vault:0.6.4)
Tagging vault:0.6.4

$ test/run.sh vault:0.6.4
testing vault:0.6.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build vault:0.6.5
Using bashbrew/cache:86499bb9fecb1e8099fec465b417878f89fa61d8added404f99c4c73ec43c631 (vault:0.6.5)
Tagging vault:0.6.5

$ test/run.sh vault:0.6.5
testing vault:0.6.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build vault:0.7.0
Using bashbrew/cache:f670752d8addbe7f4f6888fdf5f98c19cd7336c0692a2cdd09115ade3f4a68cb (vault:0.7.0)
Tagging vault:0.7.0

$ test/run.sh vault:0.7.0
testing vault:0.7.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build vault:0.7.2
Using bashbrew/cache:ff79a46b49d92aba17b14f081284074b816d56108b889b546b47802d6e8c53e5 (vault:0.7.2)
Tagging vault:0.7.2

$ test/run.sh vault:0.7.2
testing vault:0.7.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build vault:0.7.3
Using bashbrew/cache:e74fa67a6f01e8e4e52f0e42fb1bc83073c6375bb84c68b6be506d01ff58493f (vault:0.7.3)
Tagging vault:0.7.3
Tagging vault:latest

$ test/run.sh vault:0.7.3
testing vault:0.7.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build vault:0.8.0-rc1
Building bashbrew/cache:0e0230ec9f5cf9caf772ccb165fbb591318b5e695d38361f002a91da840b32db (vault:0.8.0-rc1)
Tagging vault:0.8.0-rc1

$ test/run.sh vault:0.8.0-rc1
testing vault:0.8.0-rc1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**